SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -0,0 +1,69 @@
+## Cross-server RPCs
+
+### Overview
+
+In Unreal’s native client-server architecture, your game server holds the canonical state of the whole game world. As there is a single game server, it has authority over all server Actors and so it is able to invoke and execute functions on Actors unhindered. 
+
+In SpatialOS games, there can be more than one server; these multiple servers are known as “server-workers”. (Find out more about server-workers as well as “client-workers” in the [glossary]({{URLRoot}}/content/glossary#workers).)  
+As a SpatialOS game runs across many server-workers, SpatialOS server-workers have the concept of “worker authority” - where only one server-worker at a time is able to invoke and execute functions on Actors. (Find out more about authority in the [glossary]({{URLRoot}}/content/glossary#authority).)
```

Comment:
Just had a quick skim through the doc again and I spotted that this line (#8) either needs to become part of line 7, or have an extra blank line before it.

![image](https://user-images.githubusercontent.com/32169118/47656882-f1e22e00-db87-11e8-9287-7ac072169e4c.png)